### PR TITLE
Update db_structure.sql because it's cleaner

### PR DESCRIPTION
**Pull request by Gruttersshop**
Update db_structure.sql because it's cleaner and clear what what is. Siret = Siret Ape =Ape  vat_number = vat_number  DNI=DNI and IBAN=IBAN and don't sell prestashop short by stuffing it all into DNI. or Siret or even vat into siret??? No let''s keep a clean shop and don't put it into a box in the corner of the shop. Because the are people and corporations also looking and the see it or see the box's in the shop laying around.

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 8.0.x
| Description?      | Editing Siret and Ape and Vat and Iban and at last DNI. Because now it stuffed into sometimes all in 1. It''s clear what , what is and you can also translate better. 
| Type?             | improvement / new feature / 
| Category?         | FO / BO / DB
| BC breaks?        |  no
| Deprecations?     |  no
| How to test?      | Because a testes it in 8.0 and in 1.7.8.8 by changing only this and the shop / store works as there is no problem but had to make translations because it's clear what what is. but every body can do this in BO. Because it is automated. 
| Fixed ticket?     | 
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related . It''s upgradable if you take it into the SQL or update installation. because it's the hole installation of database.  Like i did the test by 8.0 and update came 8.0.1 and poef a had to manual go to phpmyadmin and insert the rows. and it works again. 
------------------------------------------------------------------------------------------------------------------------------------
BO (back office) 

Store information

-------------------------------------------------------------------------------------------------------------------------------------------
FO (front Office)  Left is [Contact us] store information that you click by footer. Middel is all page''s and right is also by footer if you open [Stores] Is it not nice and even translated. You can translate to all over the world and the know what what is. and not like must i give my id number. I will buy some where els. !

Store information front

-----------------------------------------------------------------------------------------------------------------------------------

FR: APE Use in (presta shop like fo )
  
EN: Principal Business Activity 
NL: NACE-code (Nomenclature of Economic Activities) of SBI-code:Standaard Bedrijfsindeling
IT: Attività Principale dell'Impresa
SP: APE "CNAE" (Clasificación Nacional de Actividades Económicas).
DE: APE "NACE" "Klassifikation der Wirtschaftszweige" or "Wirtschaftszweigsystematik"

EN: VAT number (Value Added Tax number) (used FO in Prestashop B2B ) 
NL: Btw-nummer
DE: USt-IdNr. (Umsatzsteuer-Identifikationsnummer)
FR: Numéro de TVA (Numéro d'identification fiscale)
IT: Partita IVA (Partita Identificazione Fiscale)
SP: Número de IVA (Número de identificación fiscal)

FR: Siret  ( also FO in presta   but a little edit in BO by note like trow here your junk why should a B2B shop only have the costumer DNI because somewhere it''s all cooperated to DNI)
FR: Siret  
NL: KvK nummer
German: Handelskammer-Nummer
English: Chamber of Commerce number
French: numéro de la Chambre de commerce
Italian: numero della Camera di Commercio
Spanish: número de la Cámara de Comercio

SP: DNI   (it''s spanis in FO) 
In Dutch, "Nationaal identiteitsbewijs".BSN-nummer: 
In German, "Personalausweis".
In English, "National identity card".
In French, "Carte nationale d'identité".
In Italian, "Carta d'identità".
In Spanish, "Documento Nacional de Identidad".

(Not in prestashop but if we are writing why not show your company IBAN?) 
Dutch: Bankrekeningnummer (IBAN ):
German: Bankkontonummer (IBAN ):
English: Bank Account Number (IBAN ):
French: Numéro de compte bancaire (IBAN ):
Italian: Numero di conto bancario (IBAN ):
Spanish: Número de cuenta bancaria (IBAN ):

===================================================================

So to sum up. many is used only FO for regristration. But why doesn''t the store / Shop have a place to show it''s Siret? Ape? vat? Iban?  in FO or invoice or to input in BO?? Let''s keep it simple. it was also a simple extra editing in some place's.

### CODE REVIEW

**Comment by SharakPL** [comment]
Hi @Gruttersshop. I think you should start a topic about it in **Discussions** section first, because adding bunch of `todo`s and few columns to the database tables won't change a thing and is not a valuable addition to the project.

**Comment by Gruttersshop** [comment]
> Hi @Gruttersshop. I think you should start a topic about it in **Discussions** section first, because adding bunch of `todo`s and few columns to the database tables won't change a thing and is not a valuable addition to the project.

I can't show change if 1st the intallation database is not change. I have already started discussion. and when a change to much you blok me. So some 1 said. do it in little steps. So i start by the 1st step. But you blok it. So how to get to point i can show my work and let it run?? If you can help?? You can see in picture what it's going to look like??

**Comment by SharakPL** [comment]
@Gruttersshop I marked your PR as a draft. Feel free to add more commits to it.
Please let me know when your PR is ready for review.

**Comment by Gruttersshop** [comment]
> @Gruttersshop I marked your PR as a draft. Feel free to add more commits to it. Please let me know when your PR is ready for review. 

tank you SharaPL. I never came so far. i see now Merging is blocked
Merging can be performed automatically with 2 approving reviews.. What must i do now? Must i ask the 1st person that liked loves the idea and gave me advice to do it in little steps also a head of project to approve?

**Comment by Gruttersshop** [comment]
> @Gruttersshop I marked your PR as a draft. Feel free to add more commits to it. Please let me know when your PR is ready for review.   Tank you again ! further than @kpodemski  got me. But do i have to get people like @matks @jf-viguie on board so merging can start?  

_Originally posted by @Gruttersshop in https://github.com/PrestaShop/PrestaShop/discussions/31067#discussioncomment-4866488_

**Review by SharakPL — CHANGES_REQUESTED** [review]
> ... adding bunch of `todo`s and few columns to the database tables won't change a thing and is not a valuable addition to the project.

so if that's everything this PR is supposed to change then it can't be accepted in current state.
I advise you to check project's [documentation and contributing guides](https://github.com/PrestaShop/PrestaShop#user-content-documentation).

**Comment by Gruttersshop** [comment]
> > ... adding bunch of `todo`s and few columns to the database tables won't change a thing and is not a valuable addition to the project.
> 
> so if that's everything this PR is supposed to change then it can't be accepted in current state. I advise you to check project's [documentation and contributing guides](https://github.com/PrestaShop/PrestaShop#user-content-documentation).

So what's happening now? the bot is editing my to do out because it's irrelevant . ( was just also for my only where i made the changes) but is the bot than going to Merg like a need with my edit?

**Comment by Hlavtox** [comment]
@Gruttersshop Hi, please read the previous comment https://github.com/PrestaShop/PrestaShop/pull/31121#issuecomment-1410781209 and stop opening more github accounts and PRs.

**Comment by Gruttersshop** [comment]
> @Gruttersshop Hi, please read the previous comment [#31121 (comment)](https://github.com/PrestaShop/PrestaShop/pull/31121#issuecomment-1410781209) and stop opening more github accounts and PRs.

did you stop de merging? Because i was never so far and i'm doing what nicosomb said. Do 1 ting a time. And i only had 1 account that was blocked. this is 2de.
